pkg/signatures/cosign_sig_fetcher.go

```diff
@@ -124,6 +124,14 @@ func (c *cosignSignatureFetcher) FetchSignatures(ctx context.Context, image *sto
 				fullImageName, err)
 		}
 
+		var rekorBundle []byte
+		if signedPayload.Bundle != nil {
+			rekorBundle, err = json.Marshal(signedPayload.Bundle)
+			if err != nil {
+				log.Errorf("Error during marshalling rekor bundle for image %q: %v", fullImageName, err)
```

Comment:
does it make sense to store `cosignSignatures` when we could not marshal `rekorBundle`? Do we handle missing bundles

Reply:
This is a good question. I decided to keep going here and store the signature without the bundle because then signature integrations that do not verify the bundle can still succeed. When the signature is missing a bundle and the integration requires tlog verification, then verification will fail.